docs/chapters/04_features.md

```diff
@@ -773,6 +773,36 @@ export class GetProductsCount {
 
 
 
+#### Using pagination
+
+The current Booster GraphQL API includes an alternative type for your read models that stands for `List{"your-read-model-name"}`, which includes the functionality to work with pagination.
+
+The Read Model List type includes some new parameters that can be used on queries:
+ * `limit`; an integer that specifies the maximum number of items to be returned.
+ * `afterCursor`; an object that includes the key of the last retrieved item, so the next query will return the next page of results.
+
+ Example:
+```graphql
+query {
+  ListProductReadModels
+  (
+    limit: 1,
+    afterCursor: { id: "last-page-item"}
+  ) {
+    id
+    sku
+    availability
+    price
+  }
+}
+```
+
+Besides the parameters, this new type also returns a different result than the regular one. Called `{your-read-model-name}Connection`, it includes the following properties:
+ * `afterCursor`; if there are more results to paginate, it will return the object to pass to the `cursor` parameter on the next query. If there aren't more items to be shown, it will be undefined.
+ * `items`; the list of items returned by the query, if there aren't any, it will be an empty list.
+
+_Note: currenty it just works with Read Models on the AWS Provider_
```

Comment:
I wouldn't include this sentence. There are many more things that only work with AWS Provider, but it is not said. I'm saying this to avoid the need to go over all the places where we say this thing when we add this feature to the local provider, for example.

We already say at the beginning of the documentation that AWS is full supported and the others are experimental.